substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/FallbackFeature.java

```diff
@@ -167,7 +169,11 @@ public FallbackFeature() {
             addCheck(Proxy.class.getMethod("newProxyInstance", ClassLoader.class, Class[].class, InvocationHandler.class), this::collectProxyInvokes);
 
             addCheck(System.class.getMethod("loadLibrary", String.class), this::collectJNIInvokes);
-        } catch (NoSuchMethodException e) {
+
+            Class<?> generatorClass = Class.forName(Package_jdk_internal_reflect.getQualifiedName() + ".MethodAccessorGenerator");
+            Method generateMethod = generatorClass.getMethod("generateSerializationConstructor", Class.class, Class[].class, Class[].class, int.class, Class.class);
+            addCheck(generateMethod, this::collectSerializationInvokes);
```

Comment:
Class `MethodAccessorGenerator` is an implementation detail of object serialization, so the location from where it is used will not be very helpful to users. I think it would be more helpful to check for `ObjectInputStream.readObject`, `.readUnshared`, and `ObjectOutputStream.writeObject`, `.writeUnshared` (and maybe their `String` variants).

Reply:
fixed